config/locales/in_person_proofing/en.yml

```diff
@@ -126,7 +126,7 @@ en:
         zipcode: ZIP Code
     headings:
       address: Enter your current address
-      barcode: You’re ready to verify your identity in person
+      barcode: You’re ready to verify your identity in person with %{app_name}
```

Comment:
I'm surprised this didn't require any code changes to interpolate the `%{app_name}` variable? 🤔